app/presenters/admin/participant_presenter.rb

```diff
@@ -87,6 +87,22 @@ def all_induction_records
     induction_records
   end
 
+  def induction_start_date
+    if participant_profile.induction_start_date
+      participant_profile.induction_start_date.to_formatted_s(:govuk)
+    else
+      "Not yet recorded"
+    end
```

Comment:
Are we definitely using this field now? I [added some related functionality](https://github.com/DFE-Digital/early-careers-framework/pull/2634) and it hadn't been plumbed in so it was retracted.

Reply:
Yep, we’re now pulling the data from DQT and displaying it in the schools interface.